src/components/Audioplayer/index.js

```diff
@@ -395,9 +409,19 @@ export default class Audioplayer extends Component {
               onPlay={play}
               onPause={pause}
               onEnd={this.onNextAyah.bind(this)}
-            /> :
-            null
-            }
+            /> : null}
+          {isLoadedOnClient && true ?
```

Comment:
that's a remnant of development and shouldn't be there, will fix

On Mon, May 2, 2016, 9:21 PM Mohamed El Mahallawy <notifications@github.com>
wrote:

> In src/components/Audioplayer/index.js
> <https://github.com/quran/quran.com-frontend/pull/283#discussion_r61779461>
> :
>
> > @@ -395,9 +409,19 @@ export default class Audioplayer extends Component {
> >                onPlay={play}
> >                onPause={pause}
> >                onEnd={this.onNextAyah.bind(this)}
> > -            /> :
> > -            null
> > -            }
> > +            /> : null}
> > +          {isLoadedOnClient && true ?
>
> why add && true ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/283/files/032eb6c0601d65dd21dbd49fce8fb59c0c0b38cd#r61779461>
>